### DISCUSSION TITLE
Multiple entities unavailable until changed externally (e.g. Alexa app)

### DISCUSSION THREAD

**Discussion by GizmoGoody** [alandtse/alexa_media_player#1814] [Q&A]
On all of my Alexa devices, the following entities are unavailable:

- Next Alarm
- Next Reminder
- Next Timer
- Repeat switch
- Shuffle switch

The Media Player and Do not disturb switch are fine.

I've tried reloading the integration and restarting HA. Any other ideas?

**Comment by GizmoGoody** [comment]
Hmmm. It seems they respond once they're changed (e.g. Alarm set, Repeat turned on) externally (e.g. Alexa app) at which point they go from Unavailable to their expected state (e.g. DTG text, toggle, etc.). Is this working as designed or should these entities always be available regardless of their state?

**Reply by NivaBarCar** [reply]
I actually have a similiar problem. I updated HA and my alarm sensor was showing up but it seemed to be in a different time zone. So I thought I might have set a wrong time zone so I decided to re set it up via HACKS. Now the sensors are not even showing up at all.
As you write they were unavailable on your side I cannot confirm that changing it externally makes them visible on my HA.
Strange that the sensors completely disappear.
